.github/workflows/build.yml

```diff
@@ -2,7 +2,24 @@ name: 🛠️ Build + Test
 
 on:
   push:
+    paths:
+      - '*.{js,ts,cjs,mjs,d.cts,d.mts,jsx,tsx,json,jsonc}'
+      - 'build/'
+      - 'src/core'
+      - 'src/extension'
+      - 'src/module'
+      - 'bunfig.toml'
+      - 'package.json'
+
   pull_request:
+    paths:
+      - '*.{js,ts,cjs,mjs,d.cts,d.mts,jsx,tsx,json,jsonc}'
```

Comment:
I see .svg is not listed here, which means that the build action will not be triggered by a change in an svg.

One of the build steps is the `bun run check` command, which (among other things) checks that the PR is not adding an svg that doesn't have an association in the `{file|folder|language}Icons.ts` files. This is to avoid committing and shipping unused svg files. We would be missing all of those checks if there's a change in an svg. 

![image](https://github.com/user-attachments/assets/f1220122-6243-48cf-a5ab-4be05538cf5f)

Reply:
Done